featuretools/tests/entry_point_tests/utils.py

```diff
@@ -3,26 +3,39 @@
 import sys
 
 
-def import_featuretools(level=None):
-    c = ''
-    if level:
-        c += 'import os;'
-        c += 'os.environ["FEATURETOOLS_LOG_LEVEL"] = "%s";' % level
-
-    c += 'import featuretools;'
-    return python('-c', c)
-
-
-def install_featuretools_plugin():
+def get_relative_path(*args):
```

Comment:
this method name is a little vague

`join_to_entry_point_tests_module_path` would be more descriptive but perhaps not the best

Reply:
thoughts on `_get_path_to_add_on_package`?

Reply:
`_get_path_to_add_ons` and have it return `pwd + 'add-ons'` by default?

Reply:
yeah sounds good